Dockerfile

```diff
@@ -1,16 +1,15 @@
 FROM python:3.9.12
 
-ENV FLASK_APP flaskr
+ENV FLASK_APP src
+ENV FLASK_ENV development
```

Comment:
This and `FLASK_DEBUG` look like things that should be configured in the `debug` block of the architect.yml file